src/datachain/lib/dc.py

```diff
@@ -1603,18 +1601,7 @@ def filter(self, *args) -> "Self":
     @detach
     def limit(self, n: int) -> "Self":
```

Comment:
🤔 do we still need this function here as it only calls "super" method?

Reply:
This is how I found it before #206 (https://github.com/iterative/datachain/pull/206/files#diff-0b1cfd3d89c057cc57811e07eb20298eeaaee60c54cc66e5128f618d144f42eeL1601). There are a few methods in the class like this - e.g. `sample`, `shuffle`, `offset`, `filter`.

Reply:
Got it! 👍 We can address this later than, if needed.

Reply:
This makes it clear what APIs are internal (from `DatasetQuery`) and what are exposed to the user. This helps us in writing a good API reference for `DataChain` and in the future, will help us from decoupling `DatasetQuery` and `DataChain`.